llvm-external-projects/iree-dialects/test/Dialect/iree_vector_ext/roundtrip.mlir

```diff
@@ -54,19 +54,63 @@ func.func @specify_layout(%lhs: memref<32x32xf16>) -> vector<32x32xf16> {
   thread_basis   = [2, 4]
 >
 
+#nested_3 = #iree_vector_ext.nested_layout<
+  subgroups_per_workgroup = [1, 1],
+  batches_per_subgroup = [4, 2],
+  outers_per_batch = [1, 4],
+  threads_per_outer = [2, 4],
+  elements_per_thread = [4, 1],
+
+  subgroup_order = [1, 0],
+  batch_order = [1, 0],
+  thread_order = [1, 0],
+  element_order = [1, 0],
+
+  subgroup_basis = [2, 4, 8],
+  subgroup_active_ids = [true, true, false],
+  thread_basis   = [2, 4]
```

Comment:
Add cases for fully true and fully false?

Reply:
fully false is invalid so I added an invalid test for it.